packages/clarity-js/types/data.d.ts

```diff
@@ -247,7 +247,9 @@ export const enum Constant {
     UserId = "userId",
     SessionId = "sessionId",
     PageId = "pageId",
-    Mask = "•",
+    Mask = "•", // Placeholder character for explicitly masked content
+    Digit = "•", // Placeholder character for digits
```

Comment:
>• [](http://example.com/codeflow?start=13&length=1)

Could we use a digit instead?

Reply:
I will tackle this in a follow on change. Currently keep it consistent with existing masking expectation.